src/generator/AutoRest.Ruby.Azure/CodeGeneratorRba.cs

```diff
@@ -66,14 +66,14 @@ public CodeGeneratorRba()
                     continue;
                 }
 
-                if( codeModel.pageModels.Any( each => each.Name.EqualsIgnoreCase(model.Name ) ) )
+                if ( codeModel.pageModels.Any( each => each.Name.EqualsIgnoreCase(model.Name ) ) )
```

Comment:
Moreover, even you haven't followed the same across your changes as well, so looks like minor formatting error :)